packages/grid/src/features/standard-table/components/StandardTableContent.tsx

```diff
@@ -18,7 +18,9 @@ export const StandardTableContent = React.memo(function StandardTableContent<
   errorLabel,
   loading,
   items,
-  noItemsLabel = "There is no data available."
+  noItemsLabel = "There is no data available.",
```

Comment:
>As shown in the Publication Manual (p. 96), the word datum is singular, and the word data is plural. Plural nouns take plural verbs, so data should be followed by a plural verb.

are?

Reply:
Wow, have never even heard it that way :D

```
Both are correct and standard in writing and speaking.

Is data singular or plural?: Usage Guide

Data leads a life of its own quite independent of datum, of which it was originally the plural. It occurs in two constructions: as a plural noun (like earnings), taking a plural verb and plural modifiers (such as these, many, a few) but not cardinal numbers, and serving as a referent for plural pronouns (such as they, them); and as an abstract mass noun (like information), taking a singular verb and singular modifiers (such as this, much, little), and being referred to by a singular pronoun (it). Both constructions are standard. The plural construction is more common in print, evidently because the house style of several publishers mandates it.

Hint: Data can be used as a singular or a plural in writing and speaking.

This data is useful.

These data have been questioned.
```